_pages/get_involved.md

```diff
@@ -87,6 +86,13 @@ the main body of the specification.
 | [BEP008](https://docs.google.com/document/d/1FWex_kSPWVh_f4rKgd5rxJmxlboAPtQlmBc1gyZlRZM) | Magnetoencephalography (MEG)               | Guiomar Niso                                      |
 | [BEP010](https://docs.google.com/document/d/1qMUkoaXzRMlJuOcfTYNr3fTsrl4SewWjffjMD5Ew6GY) | intracranial Electroencephalograpgy (iEEG) | Chris Holdgraf, Dora Hermes                       |
 
+As an extension matures, there are cases when the BEP becomes a tool rather than merged into the main body of the specification. Below is a table of BEPs that have become tools. 
+
+| Extension label                                                                           | Title        | Moderators/leads                               | Tool name                                               |
+|-------------------------------------------------------------------------------------------|--------------|------------------------------------------------|---------------------------------------------------------|
+| [BEP015](https://docs.google.com/document/d/1WYOTXDB7GzlHoWqLjd45I3uGBgPxXddST-NTqBnroJE) | Mapping file | Eric Earl, Camille Maumet, and Vasudev Raguram | [File mapper](https://github.com/DCAN-Labs/file-mapper) |
+
+
 All of the extension ideas that are not backwards compatible and thus
 will have to wait for BIDS 2.0 are listed
 [here](https://docs.google.com/document/d/1LEgsMiisGDe1Gv-hBp1EcLmoz7AlKj6VYULUgDD3Zdw).
```

Comment:
I think we should move this sentence about BIDS 2.0 up to the top:

```
You can also contribute to ongoing BIDS Extension Proposals. Below is a
table of currently-active BEPs. The "Extension label" column provides a
direct link to the documentation.

Note that all extension ideas that are not backwards compatible and thus
will have to wait for BIDS 2.0 are listed
[here](https://docs.google.com/document/d/1LEgsMiisGDe1Gv-hBp1EcLmoz7AlKj6VYULUgDD3Zdw)
instead.

```

Reply:
perhaps to keep this focused on the BEP update may you please open a new PR with this proposal?

Reply:
Yes, I'll do that once this PR is merged. Before we can merge it, can you answer my comment in https://github.com/bids-standard/bids-website/pull/88#pullrequestreview-347202380 ?

--> it'd perhaps make the review easier :-)